src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -160,7 +160,39 @@ private void testEditSessionAction() throws Exception {
         feedbackEditPage.reloadPage();
         feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackEditSuccess.html");
 
-
+        ______TS("test two 'change' links' functionality to expand uncommon settings panels");
+        
+        // test uncommon settings for 'send emails'
+        feedbackEditPage.clickEditUncommonSettingsSendEmailsButton();
+        feedbackEditPage.verifyEditSessionBoxIsEnabled();
+        feedbackEditPage.toggleClosingSessionEmailReminderCheckbox();
+        feedbackEditPage.clickSaveSessionButton();
+        feedbackEditPage.reloadPage();
```

Comment:
What I had in mind was something like, retry 5 times with 2 second intervals between tries.